test/geo/t_spatial.clj

```diff
@@ -254,3 +254,20 @@
                         (map dist)
                         (filter (partial > 50))
                         count) => (roughly 250 10))))))
+
+(facts "features"
+       (let [g1 (s/jts-point 0 0)
+             p1 {:name "null"}
+             f1 (s/->Feature g1 p1)
```

Comment:
Does this function get defined automatically by `defrecord`?

Reply:
Exactly - `->TypeName` takes the fields as sequential arguments, and `map->TypeName` lets you pass in a single map as an argument. See https://clojure.github.io/clojure/clojure.core-api.html#clojure.core/defrecord.